superset/migrations/versions/b8d3a24d9131_new_dataset_models.py

```diff
@@ -150,55 +176,66 @@ def fetch_columns_and_metrics(self, session: Session) -> None:
     Base.metadata,
     sa.Column("table_id", sa.ForeignKey("sl_tables.id")),
     sa.Column("column_id", sa.ForeignKey("sl_columns.id")),
+    UniqueConstraint("table_id", "column_id"),
```

Comment:
For people who have already run this migration, we won't get anything new that's added in this update. Can we instead create a new migration for any additions or changes that won't affect performance?